### PR TITLE
Add test for grace login limit

### PR DESCRIPTION
**Pull request by b3lix**

### CODE REVIEW

File: ipatests/test_webui/data_pwpolicy.py [hunk 1]
```diff
@@ -90,3 +90,24 @@
         ('textbox', 'cospriority', '4'),
     ],
 }
+
+PKEY8 = 'itest-group8'
+DATA8 = {
+    'pkey': PKEY8,
+    'add': [
+        ('combobox', 'cn', 'admins'),
```
Comment (miskopo) [inline ipatests/test_webui/data_pwpolicy.py]: one last thing - you need to change `admins` here to PKEY8 as well

**Comment by miskopo** [comment]
Hello @b3lix , the fix for the failing test is now merged in upstream, so you can rebase your PR, close this one and raise it again against the official freeipa project. Thank you for your cooperation, it has been great success!

**Comment by b3lix** [comment]
Hello my friend @miskopo , happy for that fix, this PR still needs implementation of user grace login limit. Thank you also, and the Great success label is my favourite since now.

**Comment by miskopo** [comment]
Dear Erik, your new test didn't execute as it is missing its definition in the temp_commit

**Comment by b3lix** [comment]
Dear Michal, thank you for checking that, I once again forgot to change that file. We should have a counter for me doing that same thing again. :D

**Pull request closed without merging**